cds_ils/importer/eitems/importer.py

```diff
@@ -168,9 +178,9 @@ def _should_import_eitem_by_priority(self, matched_document):
 
         comparison_list = []
         for eitem in existing_eitems:
-            is_imported_or_created = self._is_imported(
-                eitem
-            ) or self._is_manually_created(eitem)
+            is_imported_or_created = self._is_imported(eitem) \
```

Comment:
I think we can completely remove this check, since we have always imported, created manually or migrated, unless you want to leave it for "undefined" cases (which should not happen by the way, I think created_by fields were required)

Reply:
Keep it for extra safety? :)